ui/src/minard/components/Axes.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {useRef, useLayoutEffect, SFC} from 'react'
+
+import {PlotEnv, TICK_PADDING_RIGHT, TICK_PADDING_TOP} from 'src/minard'
+import {clearCanvas} from 'src/minard/utils/clearCanvas'
+
+interface Props {
+  env: PlotEnv
+  axesStroke?: string
+  tickFont?: string
+  tickFill?: string
+}
+
+export const drawAxes = (
```

Comment:
would it be possible to break this function up, or add some comments to explain what each block does?